biolink-model.yaml

```diff
@@ -89,6 +89,7 @@ prefixes:
   OBOREL: 'http://purl.obolibrary.org/obo/RO_'
   ORCID: 'https://orcid.org/'
   ORPHA: 'http://www.orpha.net/ORDO/Orphanet_'
+  os: 'https://github.com/cmungall/owlstar/blob/master/owlstar.ttl'
```

Comment:
I think the more appropriate prefix to IRI mapping would be `http://w3id.org/owlstar/`.

@cmungall Would that be appropriate?